### PR TITLE
feat: Created Homepage

### PR DESCRIPTION
**Pull request by neli-rashkova**
## Homepage and components
Created a homepage that currently does not have any functionality. The Header has a green background and an h1 element with the text conduit and a p element underneath it with the text "A place to share your knowledge". All text in the Header is centered. The body of the homepage consists of a main component and a side component. The main component consists of a navigation bar that currently has one item called "Global Feed". Under the navigation bar will be the content area. There is currently no content/articles. The side component currently only contained the text "Popular Tags". Attached are two screenshots of the homepage viewed as a wider screen and as a smaller screen.

Homepage-large-screen

Homepage-small-screen

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Header() {
```
Comment (YoussefLaunchUp) [inline src/components/Header.js]: Lets rename this  component to something more descriptive like Banner.

File: src/components/BodyNavBar.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function BodyNavBar() {
```
Comment (YoussefLaunchUp) [inline src/components/BodyNavBar.js]: Lets rename this  component to something more descriptive like FeedSelector.

File: src/components/Body.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Main from './Main';
+import Side from './Side';
+
+function Body() {
```
Comment (YoussefLaunchUp) [inline src/components/Body.js]: Lets rename this  component to something more descriptive like "Feed".

File: src/components/Side.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Side() {
```
Comment (YoussefLaunchUp) [inline src/components/Side.js]: Lets rename this  component to something more descriptive like TagSelector.

File: src/components/Main.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BodyNavBar from './BodyNavBar';
+import Articles from './Articles';
+
+function Main() {
```
Comment (YoussefLaunchUp) [inline src/components/Main.js]: Lets get rid of this component and move its content to the "Body" component. (which an earlier comment recommends renaming to Feed.)

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Great Work!
The overall structure is good, some renaming and moving of contents is required.

File: src/components/Header.js (hunk 1 above)
Comment (neli-rashkova) [reply src/components/Header.js]: Done!

File: src/components/BodyNavBar.js (hunk 2 above)
Comment (neli-rashkova) [reply src/components/BodyNavBar.js]: Done!

File: src/components/Body.js (hunk 3 above)
Comment (neli-rashkova) [reply src/components/Body.js]: Done!

File: src/components/Side.js (hunk 4 above)
Comment (neli-rashkova) [reply src/components/Side.js]: Done!

File: src/components/Main.js (hunk 5 above)
Comment (neli-rashkova) [reply src/components/Main.js]: Done!

**Review by YoussefLaunchUp — APPROVED** [review]
Fantastic first PR!

**Pull request merged**